app/lib/katello/validators/content_view_environment_org_validator.rb

```diff
@@ -0,0 +1,14 @@
+module Katello
+  module Validators
+    class ContentViewEnvironmentOrgValidator < ActiveModel::Validator
+      def validate(record)
+        environment_id = record.respond_to?(:lifecycle_environment_id) ? record.lifecycle_environment_id : record.environment_id
+        view = ContentView.where(:id => record.content_view_id).first
+        environment = KTEnvironment.where(:id => environment_id).first
+        unless view.organization == environment.organization
+          record.errors[:base] << "#{view.label} could not be promoted to #{environment.label} because the content view and the environment are not in the same organization!"
```

Comment:
this string needs to be internationalized with _().  Here's a random example showing one with multiple substitutions: https://github.com/Katello/katello/blob/master/app/models/katello/task_status.rb#L148

Reply:
Added internationalization. :thumbsup: